collector/diskstats_linux.go

```diff
@@ -33,6 +32,7 @@ const (
 
 var (
 	ignoredDevices = kingpin.Flag("collector.diskstats.ignored-devices", "Regexp of devices to ignore for diskstats.").Default("^(ram|loop|fd|(h|s|v|xv)d[a-z]|nvme\\d+n\\d+p)\\d+$").String()
+	allowedDevices = kingpin.Flag("collector.diskstats.allowed-devices", "Regexp of devices to allow for diskstats.").String()
```

Comment:
Let's get this consistent with the netdev flags. So similar like there, we should:

1. rename this to device-include
2. add new flag device-exclude
3. Prefix the description of collector.diskstats.ignored-devices with "DEPRECATED: Use collector.diskstats.device-exclude" and add .Hidden() before String()
4. Support excluding by setting either device-exclude but warn if people use the now deprecated flag

Basically: https://github.com/prometheus/node_exporter/blob/master/collector/netdev_common.go#L33-L77